controllers/operatorpipeline_controller.go

```diff
@@ -53,7 +54,7 @@ func (r *OperatorPipelineReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	reqLogger := logf.FromContext(ctx, "Request.Namespace", req.Namespace, "Request.Name", req.Name)
 	reqLogger.Info("Reconciling OperatorPipeline")
 
-	currentPipeline := &certv1alpha1.OperatorPipeline{}
```

Comment:
If we want this to remain certv1alpha1, we can add that to the golangci-lint config.

Reply:
I can make that change, first pass was just to get this working.

Reply:
If you added this to the alias, do you need to rerun golangci-lint to make this certv1alpha1 again?

Reply:
Good question. The answer is no, `golangci-lint` doesn't actually validate/change a `non alias` to an `alias` for imports that don't have an `alias`. What moving this to the `alias` file does, is that if someone does use an `alias` it will throw an error. For now though, since there isn't any conflict for `alpha1`, no `alias` is needed. 

In reality, we are just reserving this for later use by this name, say for example if we had a file with two `alpha1`'s (from two different libraries/packages) we needed to import, we should `alias` them both and then the linter would throw an error if we did not use `certv1alpha1` for our import. Hope this is clear, though I realize this might be more confusing.

Reply:
Resolving